libraries/ESP8266WebServer/src/ESP8266WebServer.cpp

```diff
@@ -341,6 +349,20 @@ void ESP8266WebServer::sendContent(const String& content) {
   }
 }
 
+void ESP8266WebServer::sendContent(const char* content, size_t contentLength) {
+  const size_t unit_size = HTTP_DOWNLOAD_UNIT_SIZE;
+
+  while (contentLength) {
```

Comment:
This implementation does not match the one for sendContent(String&), which uses chunked encoding and leaves fragmentation to the lower level code (WiFiClient). It would be nice if this sendContent(data, size) and sendContent(String) would behave the same way.